### ISSUE TITLE
Add links to more courses and resources

### ISSUE THREAD

**Issue by cansavvy** [jhudsl/intro_to_r#85]
If the students want to learn more, we can save them a lot of time by having a lot of resources linked at the end of the chapters and with all these courses and resources we've been writing and finding elsewhere, it wouldn't be too difficult to add them here too!

**Comment by cansavvy** [comment]
Could be the README but really anywhere in the course where there's a concept that's really too deep for us to cover well here we can link students to more resources should they want to learn about them more in their own time.

**Comment by carriewright11** [comment]
Yeah that sounds good - we do have resources listed in the intro lecture, and we also did a lot of sharing on slack last time as well. I think @avahoffman and I had discussed adding them to each day as well. The more resources the more places the better
